Dynatrace/dynatrace-operator
# Always add ActiveGate ports no matter the capability

## Description

This PR changes the handling of the ActiveGate ports. They will be activated no matter the enabled capability, the only requirement is that at least 1 capability has to be activated.



## How can this be tested?

- install to cluster 
- create ActiveGate DynaKube 
- service `dynakube-activegate` should be created as well as the ports on the ActiveGate container
